controlnext/models.py

```diff
@@ -187,13 +186,14 @@ def __getattr__(self, item):
 
 class UserProfile(models.Model):
     user = models.ForeignKey(User, verbose_name=_("user"))
-    grower = models.ForeignKey(GrowerInfo, verbose_name=_("grower"))
+    grower = models.ManyToManyField(GrowerInfo, verbose_name=_("grower"))
```

Comment:
Bij foreignkeys in principe altijd een expliciete `related_name` aangeven. Anders krijgt je zo'n lelijke automatisch gegenereerde `grower_set` terwijl `growers` veel netter geweest zou zijn.